data/json/requirements/butchery.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "id": "full_butchery_large",
+    "type": "requirement",
+    "//": "omit butcher quality because it is checked separately",
+    "qualities": [
+      { "id": "SURFACE", "level": 3 },
+      { "id": "ROPE", "level": 2 },
+      { "id": "BUTCHERY_RACK", "level": 2 },
```

Comment:
Apparently this quality doesn't exist

Reply:
lol woops forgot to update it all when i changed the name.